### PR TITLE
Edits to cost accounting section

### PR DESCRIPTION
**Pull request by KellyatPyrofex**
## Overview
Edits to cost accounting section. Edits the result of discussion in Warsaw and collaborated on in https://docs.google.com/document/d/1E2fiXoFYBwHanNUXglkiX4EKzlZxMm9Rwa-01V5gvkY/edit?usp=sharing.

### JIRA ticket:
None

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -291,16 +291,13 @@
...
-### As a validator, to support management of my resources I want to receive as a fee either a minimum charge for a deployment or a charge determined as a percentage of the size of the contract. The fee will be whichever is greater: the absolute minimum charge or the calculated minimum.
-### As a dApp developer, I use phlo to pay the cost to deploy smart contracts on the RChain platform.
-### As a dApp developer, I need to know the minimum cost it will take to deploy my contract. The cost will be whichever is greater: the absolute minimum charge or the calculated minimum.
-### As a dApp developer, I need to know how much it will cost to execute my contract.
-### As a dApp developer, I want the phlo cost to execute a contract a contract once it's on chain to be consistent given the asm REV to phlo price and accounting for non-determinism.
-### As a client of a contract already deployed on the blockchain, I need to know how much it will cost to execute the contract.
+### As a validator, I want to be compensated in REV for setting up, storing, running, and processing transactions. I need to be able to see a record of this compenstation and retrieve my earnings.
```
Comment (cboscolo) [inline docs/features.md]: validator -> validating node operator
Just to clarify the diff between what a human expects vs. what the software in the node expects.

File: docs/features.md [hunk 2]
```diff
@@ -291,16 +291,13 @@
...
+### As a validator, I want to receive interest in REV on my bond amount as defined by the schedule in the mint.
```
Comment (cboscolo) [inline docs/features.md]: validator -> validating node operator

File: docs/features.md (hunk 1 above)
Comment (KellyatPyrofex) [reply docs/features.md]: Fixed

File: docs/features.md (hunk 2 above)
Comment (KellyatPyrofex) [reply docs/features.md]: fixed

**Pull request merged**